knowledge/Tagging.md

```diff
@@ -1,11 +1,10 @@
 ---
 title: Tagging
 created: 2024-01-18T19:49:29
-modified: 2024-01-30T16:11:22
+modified: 2024-05-03T02:38:24
 ---
 
 # Tagging
+Every year, a "tagging ceremony" will take place directly after a meeting of the general membership. During this ceremony the open governance repo will be tagged with a GPG key on behalf of Redbrick. This key will have been signed by at least 2 members of the committee and the tagging will be witnessed by at least 2 members of the general membership. The tag will be on the commit approved by the general membership of the society at the meeting beforehand. 
 
-Every year, a "tagging ceremony" will take place. This will GPG sign the git tag on the repository. A HSM can be used to sign the commit, the witnesses from committee can be marked as contributors with username (for example `~user`) and any two other ordinary members of the society can be used as witnesses and added as reviewers (for example `~user`).
```

Comment:
does this have to be removed?

Reply:
I think the open gov shouldnt have technical details in it, thats why the docs exists